device/arista/x86_64-arista_7800r3_48cq2_lc/Arista-7800R3-48CQ2-C48/jr2-a7280cr3-32d4-40x100G.config.bcm

```diff
@@ -0,0 +1,745 @@
+dtm_flow_nof_remote_cores_region_34.BCM8869X=2
+dtm_flow_nof_remote_cores_region_35.BCM8869X=2
+dtm_flow_nof_remote_cores_region_36.BCM8869X=2
+dtm_flow_nof_remote_cores_region_37.BCM8869X=2
+dtm_flow_nof_remote_cores_region_38.BCM8869X=2
+dtm_flow_nof_remote_cores_region_39.BCM8869X=2
+dtm_flow_nof_remote_cores_region_40.BCM8869X=2
+dtm_flow_nof_remote_cores_region_41.BCM8869X=2
+dtm_flow_nof_remote_cores_region_42.BCM8869X=2
+dtm_flow_nof_remote_cores_region_43.BCM8869X=2
+dtm_flow_nof_remote_cores_region_44.BCM8869X=2
+dtm_flow_nof_remote_cores_region_45.BCM8869X=2
+dtm_flow_nof_remote_cores_region_46.BCM8869X=2
+dtm_flow_nof_remote_cores_region_47.BCM8869X=2
+dtm_flow_nof_remote_cores_region_48.BCM8869X=2
+dtm_flow_nof_remote_cores_region_49.BCM8869X=2
+dtm_flow_nof_remote_cores_region_50.BCM8869X=2
+dtm_flow_nof_remote_cores_region_51.BCM8869X=2
+dtm_flow_nof_remote_cores_region_52.BCM8869X=2
+dtm_flow_nof_remote_cores_region_53.BCM8869X=2
+dtm_flow_nof_remote_cores_region_54.BCM8869X=2
+dtm_flow_nof_remote_cores_region_55.BCM8869X=2
+dtm_flow_nof_remote_cores_region_56.BCM8869X=2
+dtm_flow_nof_remote_cores_region_57.BCM8869X=2
+dtm_flow_nof_remote_cores_region_58.BCM8869X=2
+dtm_flow_nof_remote_cores_region_59.BCM8869X=2
+dtm_flow_nof_remote_cores_region_60.BCM8869X=2
+
+mdb_profile.BCM8869X=l3-xl
+
+outlif_logical_to_physical_phase_map_1=S1
+outlif_logical_to_physical_phase_map_2=L1
+outlif_logical_to_physical_phase_map_3=XL
+outlif_logical_to_physical_phase_map_4=L2
+outlif_logical_to_physical_phase_map_5=M1
+outlif_logical_to_physical_phase_map_6=M2
+outlif_logical_to_physical_phase_map_7=M3
+outlif_logical_to_physical_phase_map_8=S2
+
+outlif_physical_phase_data_granularity_S1=60
+outlif_physical_phase_data_granularity_S2=60
+outlif_physical_phase_data_granularity_M1=60
+outlif_physical_phase_data_granularity_M2=60
+outlif_physical_phase_data_granularity_M3=60
+outlif_physical_phase_data_granularity_L1=60
+outlif_physical_phase_data_granularity_L2=60
+outlif_physical_phase_data_granularity_XL=60
+
+protocol_traps_mode.BCM8869X=IN_LIF
+
+schan_intr_enable.BCM8869X=0
+tdma_intr_enable.BCM8869X=0
+tslam_intr_enable.BCM8869X=0
+miim_intr_enable.BCM8869X=0
+schan_timeout_usec.BCM8869X=300000
+tdma_timeout_usec.BCM8869X=1000000
+tslam_timeout_usec.BCM8869X=1000000
+
+appl_enable_intr_init.BCM8869X=1
+polled_irq_mode.BCM8869X=1
+polled_irq_delay.BCM8869X=1000
+
+bcm_stat_interval.BCM8869X=1000
+
+mem_cache_enable_ecc.BCM8869X=1
+mem_cache_enable_parity.BCM8869X=1
+
+serdes_nif_clk_freq_in.BCM8869X_A0=2
+serdes_nif_clk_freq_out.BCM8869X_A0=1
+serdes_fabric_clk_freq_in.BCM8869X_A0=2
+serdes_fabric_clk_freq_out.BCM8869X_A0=1
+
+serdes_nif_clk_freq_in.BCM8869X=1
+serdes_nif_clk_freq_out.BCM8869X=bypass
+serdes_fabric_clk_freq_in.BCM8869X=1
+serdes_fabric_clk_freq_out.BCM8869X=bypass
+
+dram_phy_tune_mode_on_init.BCM8869X=RUN_TUNE
+
+dport_map_direct.BCM8869X=1
+
+pmf_sexem3_stage.BCM8869X=IPMF3
+rif_id_max=0x4000
```

Comment:
Broadcom had requested to change this attribute as well for JR2,  **rif_id_max=0x6000**
@arlakshm too faced a similar issue while chassis bring up and needed this change.

This is from 4.2.x broadcom SAI release onwards.